fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentImplicits.scala

```diff
@@ -0,0 +1,119 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.{Element, Tag, Content}
+import com.gu.facia.api.models.{ImageReplace, ImageCutout, FaciaContent}
+import org.joda.time.DateTime
+
+import scala.util.Try
+
+object FaciaContentImplicits {
+
+  implicit class FaciaContentImplicit(fc: FaciaContent) {
```

Comment:
You'd have to do an import to get the implicit too, remember.

This is absolutely two ways of doing the same thing! I strongly believe you should pick one (and only slightly less strongly believe it should be the pure version you keep). The errors in this PR are an example of why it's bad, and maintaingin both is going to be SUCH A DRAG. Imagine every time someone in the mobile team adds one that they need and every time you have to be like "oh yeah, you need to update it here as well" and they say "what?! Why are you doing the the same thing in two places this is mad?!" and you say "I don't think this is multiple ways of doing the same thing at all; it's just a different way to use the underlying methods. This is how I'm using it everyone in frontend." Obvs up to you though.